src/style.css

```diff
@@ -1,3 +1,61 @@
+html {
+  --selected-text-color: #00acdc;
+  --logo-size: 40px;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.image__link {
+  text-decoration: none;
+  height: var(--logo-size);
+  width: var(--logo-size);
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding